tests/fixtures/noqa.py

```diff
@@ -378,3 +378,40 @@ def bad_default_values(  # noqa: Z459
         raise TypeError('Second')
 except TypeError:
     print('WTF?')
+
+if some and (  # noqa: Z465
```

Comment:
There's no need in putting all of these cases inside `noqa.py`.
It only contains integration tests. One is enough.
Move other cases into unit tests.